README.md

```diff
@@ -31,6 +31,29 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalculator App
+
+The SimpleCalculator App is a sample application that is designed to perform basic arithmetic operations. The app was built using 4 options - [C# Markup](https://aka.platform.uno/csharp-markup) or [XAML]() for the UI and [MVUX](https://aka.platform.uno/mvux) or [MVVM]() for state management.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc)
+
+### Counter App
+
+Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter)
+
+### ToDo App
+
+Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)
+
+### TubePlayer App
+
+The TubePlayer  App is a sample application that leverages the Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers looking to understand how to use Uno.Extensions to create a rich, responsive video streaming service.
```

Comment:
`to create a rich, responsive video streaming service.`

I'm not sure that TubePlayer's UI is responsive, is it?

Reply:
@agneszitte did we change TubePlayer to be responsive? I thought it was locked to a fixed width or something like that?

Reply:
@kazo0 yes still locked to a fixed width. You are right we need to remove the "responsive" part here as that is not an aspect we are showcasing with this sample.

@kazo0, @nickrandolph 
I think the summary of this sample should be closer to some details we can find in the workshop overview, right ?
![image](https://github.com/unoplatform/Uno.Samples/assets/16295702/6b83994d-094d-4080-be13-62681c92587e)